docker-compose.yml

```diff
@@ -237,6 +237,7 @@ services:
       - UTILISATION_REPORT_DUE_DATE_BUSINESS_DAYS_FROM_START_OF_MONTH
       - UTILISATION_REPORT_DUE_EMAIL_SCHEDULE
       - UTILISATION_REPORT_OVERDUE_EMAIL_SCHEDULE
+      - CHANGE_STREAM_ENABLED
```

Comment:
Actually, having thought about it more, I think it's because `CHANGE_STREAM_ENABLED` is always set to `false` in this `docker-compose` file at the moment, and it's only ever set to `true` in `docker-compose.replica-set.yml`. Just give this comment a thumbs up if that's correct and I'll resolve it :)

Reply:
Yep - can't have change stream enabled if we're not in a replica set